ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/ViewVulnReport/ViewVulnReportPage.tsx

```diff
@@ -1,7 +1,227 @@
-import React from 'react';
+import React, { ReactElement, useEffect, useState } from 'react';
+import { useHistory, useParams, Link, generatePath } from 'react-router-dom';
+import {
+    PageSection,
+    Title,
+    Divider,
+    Flex,
+    FlexItem,
+    Breadcrumb,
+    BreadcrumbItem,
+    Bullseye,
+    Spinner,
+    Dropdown,
+    DropdownToggle,
+    DropdownItem,
+    DropdownSeparator,
+    Tabs,
+    Tab,
+    TabTitleText,
+    TabTitleIcon,
+} from '@patternfly/react-core';
+import { CaretDownIcon, DownloadIcon, HistoryIcon, HomeIcon } from '@patternfly/react-icons';
+
+import { vulnerabilityReportPath, vulnerabilityReportsPath } from 'routePaths';
+import useFetchReport from 'Containers/Vulnerabilities/VulnerablityReporting/api/useFetchReport';
+
+import PageTitle from 'Components/PageTitle';
+import BreadcrumbItemLink from 'Components/BreadcrumbItemLink';
+import NotFoundMessage from 'Components/NotFoundMessage/NotFoundMessage';
+import useDeleteModal from '../hooks/useDeleteModal';
+import DeleteReportModal from '../components/DeleteReportModal';
+
+export type TabTitleProps = {
+    icon?: ReactElement;
+    children: string;
+};
+
+function TabTitle({ icon, children }: TabTitleProps): ReactElement {
+    return (
+        <Flex alignItems={{ default: 'alignItemsCenter' }}>
+            {icon && (
+                <FlexItem>
+                    <TabTitleIcon>{icon}</TabTitleIcon>
+                </FlexItem>
+            )}
+            <FlexItem>
+                <TabTitleText>{children}</TabTitleText>
+            </FlexItem>
+        </Flex>
+    );
+}
 
 function ViewVulnReportPage() {
-    return <div />;
+    const history = useHistory();
+    const { reportId } = useParams();
+    const [isActionsDropdownOpen, setIsActionsDropdownOpen] = useState(false);
+
+    const { reportConfiguration, isLoading, error } = useFetchReport(reportId);
```

Comment:
I don't think I can move it up to a parent component because the parent component is the switch statement for rendering the page components. We can't define the hook within the render callback for the route. See `VulnReportingPage`. I see the benefit of pulling up the state as it can be used for both views, but considering the limitations, I don't think we'll be able to do it that way.